armory/__main__.py

```diff
@@ -99,6 +105,9 @@ def run(command_args, prog, description):
         default=False,
```

Comment:
This combination of `action`, `const`, and `default` can be replaced with `action="store_true"`, and `dest` doesn't need to be set when it's the same as the long version of the flag.

Reply:
There're some other places in this file where these changes could be made; but I don't want to spam comments all over the place